src/renderer/decoder/polygonDecoder.js

```diff
@@ -57,9 +57,19 @@ export function decodePolygon (geometry) {
         breakpoints.push(index);
     }
 
+    const verticesArrayBuffer = new ArrayBuffer(4 * index);
```

Comment:
This seems to be exactly the same code as in `lineDecoder.js`, so we have an opportunity to eliminate duplicity, importing from there.

Reply:
Yes, it's the same now... But I'm not sure that extracting this would generate a good abstraction: an easy to understand function.

Reply:
I aggree, that might be coincidental. I was just thinking on exposing the linear one and consuming it from the polygon one, but it may be "killing flies with cannons" 😄